### PR TITLE
Update upgradeOmbi

### PR DESCRIPTION
**Pull request by louis-lau**
Old script has stopped working. This is the updated script (https://github.com/Unimatrix0/update_ombi/blob/master/update_ombi.sh)

### CODE REVIEW

**Comment by louis-lau** [comment]
(you might need to edit some shit like user etc).
I don't use QB myself.

File: packages/system/upgradeOmbi [hunk 1]
```diff
@@ -1,32 +1,90 @@
 #!/bin/bash
```
Comment (JMSDOnline) [inline packages/system/upgradeOmbi]: Add back user= function
```
user=$(cat /srv/rutorrent/home/db/master.txt)
```

File: packages/system/upgradeOmbi [hunk 2]
```diff
@@ -1,32 +1,90 @@
...
+##  The service file's full path  ##
 ombiservicefile="/etc/systemd/system/$ombiservicename.service"
+
+##     Ombi install directory     ##
 defaultinstalldir="/opt/Ombi"
-defaultuser="${user}"
-defaultgroup="${user}"
+
+##   User and Group Ombi runs as  ##
+defaultuser="ombi"
```
Comment (JMSDOnline) [inline packages/system/upgradeOmbi]: Address the defaultuser variable to be that of our primary user
```
defaultuser="${user}"
```

File: packages/system/upgradeOmbi [hunk 3]
```diff
@@ -1,32 +1,90 @@
...
+defaultgroup="nogroup"
```
Comment (JMSDOnline) [inline packages/system/upgradeOmbi]: Address the defaultgroup variable to be that of our primary usergroup
```
defaultgroup="${user}"
```

**Comment by JMSDOnline** [comment]
Thank you @louis-lau for submitting this to me 😃

**Pull request merged**